07-routing-gossip.md

```diff
@@ -106,13 +107,20 @@ A recipient node:
       - MUST ignore it.
 
 
-### Rationale
+## Rationale
 
 The reason for allowing deferring of a premature announcement_signatures is
 that an earlier version of the spec did not require waiting for receipt of
 funding locked: deferring rather than ignoring it allows compatibility with
 this behavior.
 
+
+# Gossip Messages
+
+Certain messages are used to relay information about the state of the
+network, and are not merely generated to send to direct peers.  We
+refer to these as "gossip messages".
```

Comment:
it might be useful to add two entries to BOLT 00 Introduction in the glossary.


* #### *Lightning message*:
   * An encrypted message sent between connected *[peers](#peers)*.
   * Foundation of the transport layer of the communication network (defined in BOLT 08)


* #### *Gossip message*:
   * Certain types of Lightning Messages that are forwarded to other peers.